Makefile

```diff
@@ -42,7 +42,8 @@
 
 ############################## TOOLCHAIN ###############################
 #
-CSC         = mcs -sdk:4.0
+SDK        :=
```

Comment:
I was told that `?=` should be used for parametrizable variables, compare https://github.com/OpenRA/OpenRA/pull/10935/files#diff-b67911656ef5d18c4ae36cb6741b7965R66 and following, but honestly I don't know why.